integration_tests/pages/education-support-plan/reviewExistingChallengesPage.ts

```diff
@@ -1,6 +1,6 @@
 import Page from '../page'
 
-export default class ReviewExistingStrengthsPage extends Page {
+export default class ReviewExistingChallengesPage extends Page {
```

Comment:
Correct previous copy and paste typo error
(has only come to light now in this PR, now that we are making use of this page class in the cypress tests)